specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -4408,6 +4499,92 @@
         }
       },
       "description": "The result of the List migrationConfigurations operation."
+    },
+	"Subnet": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID of Virtual Network Subnet"
+        }
+      },
+      "description": "Properties supplied for Subnet"
+    },
+    "NWRuleSetIpRules": {
+      "x-ms-client-flatten": true,
+      "properties": {
+        "ipMask": {
+          "type": "string",
+          "description": "IP Mask"
+        },
+        "action": {
+          "type": "string",
+          "description": "The IP Filter Action",
+          "enum": [
+            "Allow"
```

Comment:
out of curiosity, do you really want a single valued enum?

Reply:
for now, yes. the intention is to support Deny in future.